test/mocha/10-documents.js

```diff
@@ -1,36 +1,26 @@
 const {requireUncached, isRegistration} = require('./helpers');
 const {documents} = requireUncached('bedrock-tokenization');
-const {X25519KeyPair} = require('x25519-key-pair');
-const {Cipher} = require('minimal-cipher');
+const {X25519KeyAgreementKey2020} =
+  require('@digitalbazaar/x25519-key-agreement-key-2020');
+const {Cipher} = require('@digitalbazaar/minimal-cipher');
 const cipher = new Cipher();
 
 // this is test data borrowed from minimal-cipher
-const recipients = [
-  {
-    header: {
-      kid: 'did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoA' +
-        'nwWsdvktH#z6LSbysY2xFMRpGMhb7tFTLMpeuPRaqaWM1yECx2AtzE3KCc',
-      alg: 'ECDH-ES+A256KW',
-    }
-  }
-];
-
-const key1 = new X25519KeyPair({
-  id: 'did:key:z6MkgDEDniwkugeRADbi5CmHFB2eFdFKh6gSCYHFUeHXaV2x' +
-    '#z6LScXou54NfzNThVwG1aF85TCuFbVPmKuKspufF7eVmHW7G',
-  controller: 'did:key:z6MkgDEDniwkugeRADbi5CmHFB2eFdFKh6gSCYHFUeHXaV2x',
-  type: 'X25519KeyAgreementKey2019',
-  publicKeyBase58: 'rdjYkZotujxQYtF3bc88cgmkLredJ9iwvwZdBrEa8LW',
-  privateKeyBase58: '4HKArAGZaGzwutAEjsbTSjbKDLrQJAP3zLPoZQtHxeuh'
+const key1 = new X25519KeyAgreementKey2020({
+  id: 'did:key:z6MkwLz9d2sa3FJjni9A7rXmicf9NN3e5xgJPUmdqaFMTgoE#' +
+    'z6LSmgLugoC8vUoK1ouCTGKdqFdpg5jb3H193L6wFJucX14U',
+  controller: 'did:key:z6MkwLz9d2sa3FJjni9A7rXmicf9NN3e5xgJPUmdqaFMTgoE',
+  type: 'X25519KeyAgreementKey2020',
+  publicKeyMultibase: 'z6LSmgLugoC8vUoK1ouCTGKdqFdpg5jb3H193L6wFJucX14U',
+  privateKeyMultibase: 'z3wedGgRfySXFenmev8caU3eqBeDXrzDsdi21ofMZN8s8Exm'
 });
-
-const key2 = new X25519KeyPair({
-  id: 'did:key:z6MkrefS4sDAGNBdo7CeXKh52sBfK94NGMANfHKfbYpvPz8S' +
-    '#z6LScKCBLkDApcTvYPbjQi6EDKgpYwWiM9Ppd6X1PbjXF2dg',
-  controller: 'did:key:z6MkrefS4sDAGNBdo7CeXKh52sBfK94NGMANfHKfbYpvPz8S',
-  type: 'X25519KeyAgreementKey2019',
-  publicKeyBase58: 'e21pSQJj9kBT1Dxt4aGtjULhnybeYDfk7oKu95zXerv',
-  privateKeyBase58: 'bcB3uZng7RPz7VSEJSid54cyiU2STGHk4Ub91VEenPP'
+const key2 = new X25519KeyAgreementKey2020({
+  id: 'did:key:z6MkttYcTAeZbVsBiAmxFj2LNSgNzj5gAdb3hbE4QwmFTK4Z#z6LSjPQz' +
+    '1GARHBL7vnMW8XiH3UYVkgETpyk8oKhXeeFRGpQh',
```

Comment:
Please do the line break at the `#` hash fragment which will be consistent with what was done on L10.  This would be a logical way to extend long DID URLs across multiple lines.

Reply:
Updated.